content/applications/essentials/activities.rst

```diff
@@ -44,13 +46,13 @@ Schedule Activity form <activities/form>`.
 Kanban view
 -----------
 
-Another way that activities are created is from the Kanban view in an application.
+Activities can also be created from the Kanban view.
```

Comment:
Suggesting instead of the below tip admonition, as this is a bit more streamlined :)
```suggestion
Activities can also be created from the :icon:`oi-view-kanban` :guilabel:`(Kanban)` view.
```